docs/data_storage.md

```diff
@@ -0,0 +1,124 @@
+| 58                       |  58        | 2g        |      TBD                     |       TBD                 |       TBD                     |       TBD                  |
+
+
+## LUMI-F
+LUMI-F has 58 OST (Object Storage Targets), which allows for optimum performance when stripe count is set to a 58 or a multiple of 58 ( see [](https://wiki.lustre.org/images/b/b3/LUG2019-Lustre_Overstriping_Shared_Write_Performance-Farrell.pdf)
+
+| Number of parallel tasks |stripe count|stripe size| read `(GBytes/sec)` per task | read `(GBytes/sec)` total | write `(GBytes/sec)` per task | write `(GBytes/sec)` total |
+|--------------------------|------------|-----------|------------------------------|---------------------------|-------------------------------|----------------------------|
+| 1                        |  1         | 1m        |      1.71                    |      1.71                 |       1.38                    |       1.38                 |
+| 1                        |  1         | 2m        |      1.71                    |      1.71                 |       1.38                    |       1.38                 |
+| 1                        |  1         | 4m        |      1.71                    |      1.71                 |       1.38                    |       1.38                 |
+| 2                        |  2         | 2m        |                              |                           |                               |                            |
+| 4                        |  4         | 2m        |                              |                           |                               |                            |
+| 8                        |  8         | 4m        |                              |                           |                               |                            |
+| 16                       |  16        | 4m        |      1.71                    |       22.7                |       22.3                    |       22.3                 |
+| ...                      |  ...       | 4m        |                              |                           |                               |                            |
+| 32                       |  32        | 4m        |      TBD                     |       TBD                 |       TBD                     |       TBD                  |
+| 58                       |  58        | 4m        |      TBD                     |       TBD                 |       TBD                     |       TBD                  |
+
+
+
+
+## RAM FS
+
+For tasks which can make use of a non-distributed filesystem, on a single node, RAM can be effectively utulized through the RAM FS /tmp mountpoint. Obviously, RAM FS is the fastest of the storage options, but has the serious drawback of not being distributed, but rather localized to a single node. For completeness we do include the banchmark speeds in the standard-g partition.:
```

Comment:
Would you be able to provide a bash script (and a modified python script if necessary) that shows how RAM FS can be utilized?